### PR TITLE
Initial add of the wol plugin

### PR DESCRIPTION
**Pull request by The-Michael-R**
I would like to contribute a small plugin that allows to monitor a specific server by ping a configurable IP address and on user-interaction wake the server.

Plugin:
image

Configuration:
image

### CODE REVIEW

File: Tools/wol/wol-nas.10s.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#  <xbar.title>Wake on LAN (WoL)</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Michael R.</xbar.author>
+#  <xbar.author.github>The-Michael-R</xbar.author.github>
+#  <xbar.desc>Small xbar plugin to wake one server/computer using a WoL packet. Needs https://github.com/jpoliv/wakeonlan (brew or place the file wakeonlan to /urs/local/bin</xbar.desc>
+#  <xbar.image>https://github.com/The-Michael-R/xbar-plugins/Tools/wol/wol_xbar.png</xbar.image>
```
Comment (matryer) [inline Tools/wol/wol-nas.10s.sh:8]: This image is not found https://github.com/The-Michael-R/xbar-plugins/Tools/wol/wol_xbar.png

**Review by matryer — CHANGES_REQUESTED** [review]
Nice job, please update the image. :)

**Comment by The-Michael-R** [comment]
Thanks. I hope that the updated link works now.

**Comment by matryer** [comment]
@The-Michael-R I think it needs to be `https://github.com/The-Michael-R/xbar-plugins/blob/master/Tools/wol/wol_xbar.png?raw=true`?

**Comment by matryer** [comment]
@The-Michael-R Great - except we don't want the image in the PR, just linked to in the `metadata` :) Sorry to be a pain.

**Comment by The-Michael-R** [comment]
No problem. That's your repro and your rules. And that rule actually make sense.

Sorry for not be very experienced as that's my first PR to a _public_ repro.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**